content/en/real_user_monitoring/browser/troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+
+## Missing data
+
+If you can't see any RUM data or that data is missing for some users, here are the common causes and recommended fixes:
+
+| Common causes                                                                                               | Recommended fix                                                                                                                                                                                          |
+| ----------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Ad blockers][3] prevent the Browser RUM SDK from being downloaded or from sending the data to Datadog.     | Some ad blockers extend their restrictions to performance and marketing tracking tools. We recommend [installing the Browser RUM SDK via npm][4] and [forwarding the collected data through a proxy][5]. |
+| Network rules or VPN prevent the Browser RUM SDK from being downloaded or sending data to Datadog. | You may need to grant access to the endpoints required to download the SDK or to send data. The list of endpoints is available in our [CSP documentation][6].                                        |
+
+You may also want to check our [Content Security Policy guidelines][6] to make sure your website successfully grants access to the Browser RUM SDK CDN and the intake endpoint.
+
+### Check that the Browser RUM SDK is initialized
+
+Run the `window.DD_RUM.getInternalContext()` command in your browser console and check that an `application_id`, `session_id`, and view object are returned:
+
+{{< img src="real_user_monitoring/browser/troubleshooting/success_rum_internal_context.png" alt="Successful get internal context command">}}
+
+If the SDK is not installed, or if it is not successfully initialized, you may see the `ReferenceError: DD_RUM is not defined` error like the one below:
+
+{{< img src="real_user_monitoring/browser/troubleshooting/error_rum_internal_context.png" alt="Error get internal context command">}}
+
+You can also check your browser developer tools console or network tab if you notice any errors related to the loading of the Browser RUM SDK.
+
+### Check that data is sent to the Datadog intake
+
+The Browser RUM SDK sends batches of data periodically to the Datadog intake. You should see network requests targeting `/v1/input` (the URL origin part may differ due to RUM configuration) in the Network section of your browser Developer Tools:
```

Comment:
thought: is it documented somewhere when we send batches? it could be interesting to have that somewhere

Reply:
Yes definitely! Adding this to my to-do :)